src/main/kotlin/com/criptext/mail/utils/ServerErrorCodes.kt

```diff
@@ -0,0 +1,6 @@
+package com.criptext.mail.utils
+
+object ServerErrorCodes{
+    const val DeviceWasRemoved = 401
+    const val PasswordHasChanged = 403
```

Comment:
I would rather use a regular name: `Unauthorized` and `Forbidden`

Reply:
It's also changed in the next PR.